README.md

```diff
@@ -1,85 +1,155 @@
 [![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/6BOvYMwN)
 
-# Software Requirements
+# Assignment 4 - Containerization
 
-Download and install these software if you do not have them locally.
+We used Docker to containerize our various micro-services which serves our combined application.
+
+The combined application contains qualities of:
+* Question Repository Application
+* User Profile Management Application
+
+We have various micro-services including:
+* MySQL Database
+* User Service
+* Auth Service
+* Question Service
+* Frontend Service (Web UI)
+
+Additional Notes:
+
+If you are not using Windows OS, and would like a equivalent instruction for a differnt OS, do reach out to us for clarification.
+
+**For clarifications, do leave your questions at [Feedback PR](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g32/pull/1) created in our repository.**
+
+# Requirements
+
+## Network
+
+Internet connection is required.
 
-- [NodeJS](https://nodejs.org/en/download)
-- [MySQL](https://dev.mysql.com/downloads/mysql/)
+**[ IMPORTANT! ]** Do _not_ test while on NUS grounds or connected
+(either directly or indirectly) to NUS Wifi.
+(NUS networks blocks MongoDB which is required by our application.) 
 
-For development, you may also want to install:
+## Software
+Download and install these software if you do not have them locally.
 
-- [MongoDB Compass](https://www.mongodb.com/try/download/compass)
 - [Docker Desktop](https://www.docker.com/get-started/)
-- [Postman](https://www.postman.com/downloads/)
 
-# Setup
+# Testing
 
-## Setup MySQL locally
+For clarity, the commands given below should be executed at the root directory if not specified otherwise.
 
-1. Search for **MySQL 8.1 Command Line Client** on your computer. Execute it to open up the terminal.
-2. Enter your root password.
-3. Run the SQL statements in `./User/schema.sql` on the terminal.
+_The commands below are provided for convenience and may be incorrect.
+If reading the textual instruction is not clear enough, do reach out to us for clarification._
```

Comment:
Will rephrasing these line be better? 
e.g.
"The commands below are provided for convenience and may be incorrect.
~~If reading the textual instruction is not clear enough, do reach out to us for clarification.~~ Do reach out to us for clarification if needed."